.github/workflows/test-notebooks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "Using compiler: $1"
+for notebook in examples/cfd examples/seismic/tutorials examples/compiler examples/userapi
```

Comment:
As I can see from the log it seems that this for loop only evaluates examples/cfd notebooks.
Maybe something needs to change in this line.

Reply:
From this [log output](https://github.com/devitocodes/devito/commit/a6cc03acdf8810a7da5162c155e43328458c4379/checks?check_suite_id=394956119) I can see that it does indeed evaluate all notebooks listed in the for loop with:
- examples/cfd -->
```
============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/runner/work/devito/devito, inifile: setup.cfg
plugins: nbval-0.9.4, cov-2.8.1
collected 76 items

examples/cfd/01_convection.ipynb ..........                              [ 13%]
examples/cfd/01_convection_revisited.ipynb ..........                    [ 26%]
examples/cfd/02_convection_nonlinear.ipynb ..............                [ 44%]
examples/cfd/03_diffusion.ipynb .........                                [ 56%]
examples/cfd/03_diffusion_nonuniform.ipynb .........                     [ 68%]
examples/cfd/04_burgers.ipynb ...........                                [ 82%]
examples/cfd/05_laplace.ipynb .........                                  [ 94%]
examples/cfd/06_poisson.ipynb ....                                       [100%]
```
- examples/seimic/tutorials/ -->
```
============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/runner/work/devito/devito, inifile: setup.cfg
plugins: nbval-0.9.4, cov-2.8.1
collected 149 items

examples/seismic/tutorials/01_modelling.ipynb ............               [  8%]
examples/seismic/tutorials/02_rtm.ipynb ............                     [ 16%]
examples/seismic/tutorials/03_fwi.ipynb .............s.s                 [ 26%]
examples/seismic/tutorials/04_dask.ipynb .......sss                      [ 33%]
examples/seismic/tutorials/05_staggered_acoustic.ipynb ...s......s...s.  [ 44%]
examples/seismic/tutorials/06_elastic.ipynb ...s....s...s.               [ 53%]
examples/seismic/tutorials/06_elastic_varying_parameters.ipynb ..s.s..s. [ 59%]
.....s...ss..s..s                                                        [ 71%]
examples/seismic/tutorials/07_DRP_schemes.ipynb ....................     [ 84%]
examples/seismic/tutorials/08_snapshotting.ipynb ........s               [ 90%]
examples/seismic/tutorials/09_viscoelastic.ipynb ............s.          [100%]
```
- examples/compiler -->
```
============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/runner/work/devito/devito, inifile: setup.cfg
plugins: nbval-0.9.4, cov-2.8.1
collected 52 items

examples/compiler/01_data_regions.ipynb .............                    [ 25%]
examples/compiler/02_indexification.ipynb .............                  [ 50%]
examples/compiler/03_iet-A.ipynb ................                        [ 80%]
examples/compiler/04_iet-B.ipynb ..........                              [100%]
```
- examples/userapi -->
```
============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/runner/work/devito/devito, inifile: setup.cfg
plugins: nbval-0.9.4, cov-2.8.1
collected 16 items

examples/userapi/apply.ipynb ................                            [100%]
```

I think it was just a case of the output being quite verbose and hiding a lot of the other tests.

I have also noticed that sometimes it shows no output at all and I have to toggle the drop down button for " Evaluate cells of example notebooks" sometimes. This is more of an issue with Actions UI it seems.

p.s. I'm just showing a single workflow as an example. They are all similar and all pass.

Reply:
ΟΗ! NIce then! I did not notice that. Sorry for wasting your time!

Reply:
No worries! 👍